src/Components/CreateImageWizard/validators.ts

```diff
@@ -79,6 +85,41 @@ export const isSshKeyValid = (sshKey: string) => {
   return isPatternValid;
 };
 
+interface PasswordValidationResult {
+  isValid: boolean;
+  strength: HelperTextVariant;
+}
+
+const isPasswordLengthValid = (password: string): boolean => {
+  if (!password) return true;
```

Comment:
I think this is confusing because a password with length 0 violates the minimum length requirement of 6. I would expect `isPasswordLengthValid('')` to return `false`. 

I think we should handle the special case where there is no password (`password === ''`) separately.

Reply:
changed that to be with minimum 6 length